### DISCUSSION TITLE
capacity changes and parameters that influence the capacity

### DISCUSSION THREAD

**Discussion by ddagrinho** [pybamm-team/PyBaMM#3458] [Q&A]
Hello,
so if i understand correctly from previous discussions, in order to change the nominal capacity you need to change also geometrical parameters like the electrode thicknesses. You can do the same if you change the active material fraction? can you specify which parameters influence the capacity calculation or attach a link of how the capacity is getting influenced by these parameters?

thank you in advance.

**Comment by brosaplanella** [comment]
Not quite: nominal capacity is used only to convert C-rate to current. Changing the parameters you mention would affect the actual capacity but not the nominal one, this needs to be changed accordingly. The capacity of each electrode is given by 

$$Q = F A L c^{\mathrm{max}} \varepsilon_\mathrm{s} $$

where $F$ is the Faraday constant, $A$ the electrode plate area, $c^\mathrm{max}$ is the maximum concentration and $\varepsilon_\mathrm{s}$ is the active material volume fraction. The actual cell capacity is related to the capacity of each electrode but it also depends on the balancing.
